### ISSUE TITLE
Wrong button copy in Catalan

### ISSUE THREAD

**Issue by sauloperez** [coopdevs/katuma-landing-page#62]
When displaying the page in Catalan, the "contact us" button still shows `let's talk` instead of `parlem`. See below

![Screenshot from 2019-09-02 11-34-13](https://user-images.githubusercontent.com/762088/64105150-ec401080-cd75-11e9-8675-588356f772de.png)

**Comment by chrysillala** [comment]
hey, may i help?

**Comment by sauloperez** [comment]
Sure @chrysillala ! and welcome to the team.

You can find the locales in https://github.com/coopdevs/katuma-landing-page/tree/master/locales/ca. It should be quite straightforward to debug the following line

https://github.com/coopdevs/katuma-landing-page/blob/e6eeaa42f6754bd6b2eed8702dcef88b1f4a9e21/index.html#L124

so it picks the translation. As you can see, we use https://www.i18next.com for i18n.

**Comment by chrysillala** [comment]
hey @sauloperez , can you help me with the setup? I cannot run `bundle install` it said command not found. thanks

**Comment by sauloperez** [comment]
Yes, sorry. You might have seen that command in the README. We assume some familiarity with Ruby in there. 

So, you need to install Ruby using the version specified in https://github.com/coopdevs/katuma-landing-page/blob/master/.ruby-version. I recommend you to do so using [rbenv](https://github.com/rbenv/rbenv#installing-ruby-versions). You'll find the steps to install a particular Ruby version in their docs.

Then, you should be able to install [bundler](https://bundler.io) by running `gem install bundler -v 1.17.1`. This will allow running the command you mentioned, which will install the dependencies of the project.

I hope it helps. Don't hesitate to ask anything!

**Comment by chrysillala** [comment]
hi @sauloperez i'm getting difficulties setting up the environment. I got this error when doing `bundle install`, tried to install eventmachine separately but still got the error. can you please help? Thanks!
![image](https://user-images.githubusercontent.com/4686015/66210795-d90dc100-e6e4-11e9-9baa-5068694141f2.png)

**Comment by sauloperez** [comment]
Hi @chrysillala I'll do my best but these specific issues are hard to solve remotely. Could you please take a look at the file mkmf.log it mentions in? You should find it in that path `/var/folders/k_/...`, upload it somewhere and share it so I can take a look.

**Comment by chrysillala** [comment]
@sauloperez hey, i tried to fix the bug without setting the env, turns out there's typo in your code, please check my PR https://github.com/coopdevs/katuma-landing-page/pull/68
